source/stylesheets/refills/_search-tools.scss

```diff
@@ -54,7 +54,7 @@
     }
 
     > div {
-      @include box-sizing(border-box);
+      box-sizing: border-box;
```

Comment:
Properties should be ordered background-color, border, border-radius, box-sizing, font-size, margin-bottom, padding, position, width